src/emulator/gui/src/gui.cpp

```diff
@@ -166,7 +166,7 @@ void init_display(GuiState &gui) {
     glGenTextures(1, &gui.display);
 }
 
-static void init_icons(GuiState &gui, HostState &host) {
+void init_icons(GuiState &gui, HostState &host) {
     for (Game &game : gui.game_selector.games) {
```

Comment:
For this i thinks other things

Reply:
but for clear is very bad idea, get more time for reload all etc, i have set this for game list exactly for that

Reply:
I think when you should reload all games you should reload all games. Maybe a homebrew developer changed the icon. Maybe they deleted a game from their library and you don't need the icon anymore.

I think this is the proper way to approach it.

Reply:
ah yes, for icon can change i have see that lol, but in all case, icon is changed after reboot like actually,

Reply:
You're right... but wouldn't it be easier to just press a button then to restart Vita3K? Especially when there is a button for that exact purpose. It would be very useful!